plugins/org.eclipse.reddeer.core/src/org/eclipse/reddeer/core/lookup/WidgetLookup.java

```diff
@@ -114,6 +115,17 @@ public static WidgetLookup getInstance() {
 		return (T)found.getResult();
 	}
 	
+	private String createRererencedCompositeDebugMsg(ReferencedComposite refComposite){
+		StringBuilder sb = new StringBuilder();
+
+		if (refComposite == null){
+			sb.append(" no referenced composite specified");
```

Comment:
Please change appended text to " with no referenced composite specified". So it would result in "Looking up active widget with class type some.package.Widget with no referenced composite specified...."